test/functional/combine_logs.py

```diff
@@ -78,10 +77,11 @@ def read_logs(tmp_dir):
     for each of the input log files."""
 
     # Find out what the folder is called that holds the debug.log file
-    chain = glob.glob("{}/node0/*/debug.log".format(tmp_dir))
-    if chain:
-        chain = chain[0]  # pick the first one if more than one chain was found (should never happen)
-        chain = re.search(r'node0/(.+?)/debug\.log$', chain).group(1)  # extract the chain name
+    glob = pathlib.Path(tmp_dir).glob('node0/**/debug.log')
```

Comment:
Is this recursive on purpose or would `'node0/*/debug.log'` suffice?

Reply:
Not on purpose. Non-recursive would suffice, I can't imagine a scenario where debug.log ends up in any subdirectories of node0.